e2e-tests/tfm/cypress/fixtures/constants.js

```diff
@@ -43,6 +43,34 @@ const DISPLAY_USER_TEAMS = {
   PDC_READ: 'PDC read',
 };
 
+/**
```

Comment:
This is a very nice improvement! Is it worth moving this more to an E2E/cypress readme and just leaving a comment here saying 'see readme for alias keys' etc.
This feels like something useful for someone to read through as an approach on how to use cypress that's a bit hidden away here

Reply:
I've updated the `README.md` - if you could have a look that would be great :)